example/ios/Runner/Info.plist

```diff
@@ -21,9 +21,11 @@
 	<key>CFBundleVersion</key>
 	<string>$(FLUTTER_BUILD_NUMBER)</string>
 	<key>CleverTapAccountID</key>
-	<string>TEST-65R-487-R85Z</string>
+	<string>7Z8-W9R-876Z</string>
```

Comment:
@akashvercetti remove credentials before merging

Reply:
This is done.